### ISSUE TITLE
Herbiboar Tracking

### ISSUE THREAD

**Issue by lulwut** [runelite/runelite#545]
Hi, 

I hope I'm doing this correctly, when hunting heriboars you click on various objects which create a pathway to the next object by producing paw prints until they finally lead to a nest type thing which you "attack" and gets herbs from.

My feature request is that the tiles which get the paw prints get highlighted as it can be quite hard to follow a path if the jungle is dense.

Thanks!

**Comment by Sethtroll** [comment]
@tylerthardy (also Perterter#8027 in discord) was working on this, though I haven't seen him around in a couple weeks due to IRL stuff. Maybe he will update it soon.

**Comment by lulwut** [comment]
How can I mark this as a feature request?

**Comment by tylerthardy** [comment]
I've returned. This plugin is complete, but I have to adjust some things so it does not break the game's ToS.  Coming very soon. 😉

**Comment by lulwut** [comment]
That's great, what kind of things had to be modified? Just curious. Can't wait until it's released

**Comment by deathbeam** [comment]
Fixed in #688
